DEVELOPER_GUIDE.md

```diff
@@ -230,6 +230,11 @@ $ wsl -d docker-desktop
 $ sysctl -w vm.max_map_count=262144
 ```
 
+#### Debugging
+You can debug the OpenSearch Dashboards server by
+1. Running `yarn debug`. This will start the OpenSearch Dashboards development server with a debug port open on port `9229`.
```

Comment:
I think we need to attach debugger client first right? If not, I see 9230 is picked in my vscode. If I attach it, then it is 9229

Reply:
I'm considering vscode's debugger as the client here and the server running in debug mode as the debugger server. Do you think I should reword to avoid confusion?

Reply:
Or are you saying you attach with vscode first before you run the server in debug mode?

Reply:
Gotcha. I'll update the ordering here.